protocols/smtp/src/main/java/org/apache/james/protocols/smtp/core/esmtp/AuthCmdHandler.java

```diff
@@ -387,6 +389,14 @@ protected Response doAuthTest(SMTPSession session, Username username, String pas
                     } else if (SMTPRetCode.AUTH_OK.equals(res.getRetCode())) {
                         // TODO: Make this string a more useful debug message
                         AUTHENTICATION_DEDICATED_LOGGER.debug("AUTH method {} succeeded", authType);
+
+                        AuditTrail.entry()
+                            .username(username.asString())
+                            .remoteIP(session.getRemoteAddress().getAddress().getHostAddress())
+                            .protocol("SMTP")
+                            .action("AUTH")
+                            .parameters(ImmutableMap.of("authType", authType))
+                            .log("SMTP Authentication succeeded.");
```

Comment:
Handle the fact that session.getRemoteAddress can be null (if the client disconnects)